pages/linux/ip-rule.md

```diff
@@ -0,0 +1,36 @@
+# ip rule
+
+> IP Routing policy database management.
+> More information: <https://man7.org/linux/man-pages/man8/ip-rule.8.html>.
+
+- Display the routing policy:
+
+`ip rule {{show|list}}`
+
+- Insert a new rule use the source prefix:
```

Comment:
I'm not really understanding this sentence 🤔

Reply:
If I change it to "Adding a rule to the routing policy database based on the source address of the packet", do you understand?

Reply:
Ah thanks, now I get it. But I think we can change it a little bit:
```suggestion
- Add a new rule based on the source address of the specified package:
```
Could you please also change the following examples to a similar wording?

Reply:
Okay